.travis.yml

```
@@ -24,4 +24,4 @@ install:
 
 script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]];   then hack/travis_osx.sh ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make vendor && ./hack/tree_status.sh && make check ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make vendor && ./hack/tree_status.sh && make local-cross check ; fi
```

IDK, should this be:
```suggestion
  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make vendor && ./hack/tree_status.sh && make local-cross && make check ; fi
```

---

`make` can be told to build multiple targets, and it stops at the first error whatever it's doing, so they're effectively equivalent.

---

ah, ty.

---

Since I'm rebasing anyway, taking this change.